src/main/java/br/com/dandrade/viagens/controllers/CountryController.java

```diff
@@ -0,0 +1,37 @@
+package br.com.dandrade.viagens.controllers;
+
+import br.com.dandrade.viagens.models.Country;
+import br.com.dandrade.viagens.repository.CountryRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "/api/country",
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Quando configuramos o starter da web, o default já é a serialização ser feita via jackson. Então o json é a opção padrão. Aqui nem precisa colocar essa configuração :).

Reply:
sim! foi vacilo meu.